### PR TITLE
Boy scout rule - Chapter I

### PR DESCRIPTION
**Pull request by christophsaile**
Adjusted some of the text and added some comments for improvement

### CODE REVIEW

File: src/fundamentals-1.md [hunk 1]
```diff
@@ -94,11 +94,15 @@ In this challenge you will learn how to use HTML Semantic elements and how to de
 
 #### Bring your body in shape - from header to footer:
 First create an html page based on the HTML5 Boilerplate and segment it with semantic elements.
-Then add basic CSS styling for these areas to match the layout shown in the screen graphics.
+Then add basic CSS styling for these areas (they yellow boxes) to match the layout shown in the screen graphics.
```
Comment (luke-m) [inline src/fundamentals-1.md]:
```suggestion
Then add basic CSS styling for these areas (the yellow boxes) to match the layout shown in the screen graphics.
```

File: src/fundamentals-1.md [hunk 2]
```diff
@@ -143,6 +148,9 @@ Add HTML5 compliant headlines and sections to structure the content of your page
 #### Inner values matter!
 Add dummy text and image content to your page, you can use dummy text and image generators for that.
 With latter you can easily create dummy images like this:
+
+---- Text should be above the img ----
```
Comment (luke-m) [inline src/fundamentals-1.md]: Are these `----` prefixed sentences "comments"? I'm not sure how to interpret them

Comment (christophsaile) [reply src/fundamentals-1.md]: yes they are comments, because i didnt know how to change it by myself :)

File: src/fundamentals-2.md [hunk 3]
```diff
@@ -11,15 +11,16 @@ After you built a static page in the last chapter and learned about layout, cont
...
+*Note: the command lines in this abstract are all for Bash and perhaps need some adjustments for Shell / PowerShell.*
 
 ### Step 1: Install Node.js and NPM
 
 Node.js is a way to run javascript code on your machine, without the need for a browser. It's mainly used to run servers and automatic tasks. You won't need to use it a lot, so it's not mandatory to do any work with it for this chapter.
 Node.js comes with NPM, its very own package manager. This tool is used by the majority of web developers to install packages, libraries and other tools. It's pretty much your bread and butter and it is very much obligatory that you learn how to use it. The first step is to download and install it.
 > For windows we recomment [nvm](https://github.com/creationix/nvm) for installing it.
 If you are not on windows, just install it from [the Node.js homepage](https://nodejs.org/en/).
-To ensure the installation worked correctly, type the following command into your command line:  `node -v && npm -v`
+To ensure the installation worked correctly, type the following command into your command line:  `node -v && npm -v` (for PowerShell `(node -v) -and (nqm -v)`)
```
Comment (SheepFromHeaven) [inline src/fundamentals-2.md]:
```suggestion
To ensure the installation worked correctly, type the following command into your command line:  `node -v && npm -v` (for PowerShell `(node -v) -and (npm -v)`)
```

**Pull request merged**